cli/cmd/integration.go

```diff
@@ -103,6 +105,13 @@ var (
 				return errors.New(msg)
 			}
 
+			integrationType, _ := api.FindIntegrationType(integration.Data[0].Type)
+			err = cli.LwApi.V2.Schemas.GetService(integrationType.Schema()).Get(args[0], &resp)
+			if err != nil {
+				return errors.Wrap(err, "unable to get integration")
```

Comment:
lets not error if we couldn't get the service, lets just not print the extra details